frontends/web/src/app.tsx

```diff
@@ -71,27 +72,36 @@ class App extends Component<Props, State> {
     /**
      * Gets fired when the route changes.
      */
-    private handleRoute = () => {
+    private handleRoute = async (e:RouterOnChangeArgs) => {
         if (panelStore.state.activeSidebar) {
             toggleSidebar();
         }
+
+        await sleep(0); // this must be run async
+        const {accounts} = this.state;
+        const isIndex = e.url === '/' || e.url === '/index.html' || e.url === '/android_asset/web/index.html';
```

Comment:
seems like this and the next few lines are copied from `maybeRoute()`. Would it be possible to simply call `this.maybeRoute()` here instead?